clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
hmm, right, good point. Maybe using the [copy module](https://docs.python.org/2/library/copy.html) leads to nicer code (`origCmdLine = copy.copy(cmdLine)` because it does not put focus on the type but on the copying.